lib/active_record/virtual_attributes/virtual_delegates.rb

```diff
@@ -72,7 +76,6 @@ def define_virtual_delegate(method_name, col, options)
           col = col.to_s
           type = options[:type] || to_ref.klass.type_for_attribute(col)
           type = ActiveRecord::Type.lookup(type) if type.kind_of?(Symbol)
-          raise "unknown attribute #{to}##{col} referenced in #{name}" unless type
```

Comment:
Do you have any idea what situation this was handling?  I'm fine with removing it but don't understand how the type could be something that's not known.

Reply:
Maybe we can drop this. if we detected the type but the attribute was not found, then this will be `nil`